src/main/java/appeng/client/gui/implementations/GuiCraftingCPU.java

```diff
@@ -252,7 +257,7 @@ public void drawScreen(final int mouseX, final int mouseY, final float btn) {
         final int offY = 23;
         int y = 0;
         int x = 0;
-        for (int z = 0; z <= 4 * 5; z++) {
+        for (int z = 0; z <= 4 * rows; z++) {
```

Comment:
shouln't it be (rows-1) or <= changed to < ?

Reply:
You right.. partially :) There are to many magic numbers around. Make this relevant to current rendering.